src/sidebar/util/type-coercions.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Type conversion methods that gently coerce values that other parts
+ * of the UI may make assumptions on. This is needed for incoming parameters
+ * that are otherwise not sanitized. Note that these methods do not throw 
+ * errors or act like validators with mal-formed values.
+ */
+
+/**
+ * Returns a boolean
+ *
+ * @param {any} value - initial value
+ * @return {boolean}
+ */
+function toBoolean(value) {
+  if (typeof value === 'string') {
+    if (value.trim().toLocaleLowerCase() === 'false') {
+      // "false", "False", " false", "FALSE" all return false
+      return false;
+    }
+  }
+  const numericalVal = Number(value);
+  if (!isNaN(numericalVal)) {
+    return Boolean(numericalVal);
+  }
+  // Any non numerical or falsely string should return true, otherwise
+  // return false
```

Comment:
There is a commented out line of code here.

Reply:
haha, thats the end of my sentence, wow that ended up looking wrong!